packages/cli/src/baseCommand.js

```diff
@@ -0,0 +1,58 @@
+const { Command } = require('@oclif/command');
+const { isEmpty } = require('lodash');
+const { cli: ux } = require('cli-ux');
+
+const { readCredentials, checkCredentials } = require('./utils/api');
+
+class ZapierBaseCommand extends Command {
+  _version() {
+    return 'blah blah ok'; // this should overwrite the version string, but doesn't?
```

Comment:
not sure why this doesn't work (we'd want to have it replicate what we have [here](https://github.com/zapier/zapier-platform/blob/9e776f07c8ecd21283b3f57cce1aa7de7af913ca/packages/cli/src/utils/misc.js#L158). There's an [open issue](https://github.com/oclif/oclif/issues/254) for it, so it'll be fixable.